src/server/graphql/queries/invoiceDetails.js

```diff
@@ -20,13 +20,12 @@ export default {
 
     // AUTH
     const viewerId = getUserId(authToken)
-    const [type, maybeOrgId] = invoiceId.split('_')
-    const isUpcoming = type === 'upcoming'
+    const isUpcoming = invoiceId.startsWith('upcoming_')
     const currentInvoice = await r
       .table('Invoice')
       .get(invoiceId)
       .default(null)
-    const orgId = (currentInvoice && currentInvoice.orgId) || maybeOrgId
+    const orgId = (currentInvoice && currentInvoice.orgId) || invoiceId.substring(9) // remove 'upcoming_'
```

Comment:
longstanding parse bug that affects any org with an underscore in their id :see_no_evil:

Reply:
doh